highfidelity/hifi
# Backup old content on update

- Backup old content instead of deleting it during a sandbox content update.

(UI still need some work, will cleanup tomorrow)



Test plan:
- Launch sandbox (content set should get installed).
- Close sandbox.
- Go to `C:\Users\Clement\AppData\Roaming\High Fidelity - PRXXXX\Server Console\config.json`.
- Change the homeContentLastModified value to an old date (ie: 2012 should do it).
- Start sandbox.
- You should get prompted for an update.
- Accept and notice the content set getting re-downloaded.
- There should be another window explaining how to restore the content
- Follow instructions and restore original content.
